### PR TITLE
add config item pessimistic-txn.constraint-check-in-place-pessimistic

### PR DESCRIPTION
**Pull request by ekexium**
Signed-off-by: ekexium <eke@fastmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -706,6 +706,11 @@ TiDB 服务状态相关配置。
 
 + 默认值：false
 
+### constraint-check-in-place-pessimistic
```
Comment (qiancai) [inline tidb-configuration-file.md:709]: 这是从 v6.4 引入的吗，如果是，请加一个 <span class="version-mark">从 v6.4.0 版本开始引入</span>

Comment (ekexium) [reply tidb-configuration-file.md]: done

File: system-variables.md [hunk 2]
```diff
@@ -798,9 +798,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_constraint_check_in_place_pessimistic` <span class="version-mark">从 v6.3.0 版本开始引入</span>
 
-- 作用域：SESSION | GLOBAL
+- 作用域：SESSION
 - 是否持久化到集群：是
-- 默认值：`ON`
+- 默认值：由配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic) 决定，该配置项的默认值为 `TRUE`(`ON`)。
```
Comment (qiancai) [inline system-variables.md:803]:
```suggestion
- 默认值：由配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic) 控制。当该配置项为默认值 `TRUE` 时，该变量的默认值为 `ON`。  
```

Comment (qiancai) [reply system-variables.md:803]: 该配置项为默认值 `TRUE`，这个 `TRUE` 可以小写吗，和 tidb-configuration-file.md 里保持一致

Comment (ekexium) [reply system-variables.md:803]: 不写默认值的话，还要跳到配置项页面才能知道默认值是啥也太麻烦了……
on=true 这个应该找个地方集中写而不是每个碰到的地方都单独说一下吧……

File: system-variables.md [hunk 3]
```diff
@@ -798,9 +798,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_constraint_check_in_place_pessimistic` <span class="version-mark">从 v6.3.0 版本开始引入</span>
 
-- 作用域：SESSION | GLOBAL
+- 作用域：SESSION
 - 是否持久化到集群：是
```
Comment (Oreoxmt) [inline system-variables.md:802]:
```suggestion
```
SESSION 变量不持久化到集群

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -706,6 +706,11 @@ TiDB 服务状态相关配置。
 
 + 默认值：false
 
+### constraint-check-in-place-pessimistic <span class="version-mark">从 v6.4.0 版本开始引入</span>
+
++ 用来控制系统变量 [`tidb_constraint_check_in_place_pessimistic`](/system-variables.md#tidb_constraint_check_in_place_pessimistic-从-v630-版本开始引入) 的默认值。
++ 默认值: true
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:712]:
```suggestion
+ 默认值：true
```

File: system-variables.md (hunk 2 above)
Comment (qiancai) [reply system-variables.md]: 写了的😂。 on=true 这个集中写了，也需要用户跳到一个地方去看。
image

Comment (ekexium) [reply system-variables.md]: 好吧。不过我觉得看起来很复杂

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

File: system-variables.md [hunk 5]
```diff
@@ -798,9 +798,8 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_constraint_check_in_place_pessimistic` <span class="version-mark">从 v6.3.0 版本开始引入</span>
 
-- 作用域：SESSION | GLOBAL
-- 是否持久化到集群：是
-- 默认值：`ON`
+- 作用域：SESSION
```
Comment (qiancai) [inline system-variables.md:801]:
```suggestion
- 作用域：SESSION
- 是否持久化到集群：否
```

Comment (qiancai) [reply system-variables.md:801]: @ekexium 请看看 “是否持久化到集群” 是 “否” 吗？

**Comment by qiancai** [comment]
@ekexium Would you please help find a tech reviewer? Thanks.

**Review by you06 — APPROVED** [review]

File: system-variables.md [hunk 6]
```diff
@@ -798,9 +798,8 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+- 默认值：当配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic) 为默认值 `true` 时，该变量的默认值为 `ON`。当配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic) 为 `false` 时，该变量的默认值为 `OFF`。
```
Comment (qiancai) [inline system-variables.md:802]:
```suggestion
- 默认值：当配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic-从-v640-版本开始引入) 为默认值 `true` 时，该变量的默认值为 `ON`。当配置项 [`pessimistic-txn.constraint-check-in-place-pessimistic`](/tidb-configuration-file.md#constraint-check-in-place-pessimistic-从-v640-版本开始引入) 为 `false` 时，该变量的默认值为 `OFF`。
```

**Pull request merged**